src/components/Admin/Module/DalSettings.vue

```diff
@@ -47,17 +47,33 @@
     </b-form-group>
 
     <b-form-group
-      :label="$t('system-fields.label')"
       :description="$t('system-fields.description')"
     >
+      <div class="my-4 d-flex justify-content-between flex-wrap">
+        <label>
+          {{ $t('system-fields.label') }}
+        </label>
+        <b-form-radio-group
```

Comment:
These should probably be checkboxes (switches), since each group should just toggle its fields on or off